utilities/policy_utilities.mak

```diff
@@ -55,236 +53,237 @@ AR=ar
 #export LD_LIBRARY_PATH=/usr/local/lib
 LDFLAGS= -L $(LOCAL_LIB) -lprotobuf -lgtest -lgflags -lpthread -L/usr/local/opt/openssl@1.1/lib/ -lcrypto -lssl
 
-measurement_utility_obj=$(O)/measurement_utility.o $(O)/support.o $(O)/certifier.o $(O)/certifier_proofs.o \
-$(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-make_indirect_vse_clause_obj= $(O)/make_indirect_vse_clause.o $(O)/support.o $(O)/certifier.o \
-$(O)/certifier_proofs.o $(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-make_simple_vse_clause_obj= $(O)/make_simple_vse_clause.o $(O)/support.o $(O)/certifier.o \
-$(O)/certifier_proofs.o $(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-make_unary_vse_clause_obj= $(O)/make_unary_vse_clause.o $(O)/support.o $(O)/certifier.o \
-$(O)/certifier_proofs.o $(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-make_signed_claim_from_vse_clause_obj= $(O)/make_signed_claim_from_vse_clause.o $(O)/support.o \
-$(O)/certifier.o $(O)/certifier_proofs.o $(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-print_vse_clause_obj= $(O)/print_vse_clause.o $(O)/support.o $(O)/certifier.o $(O)/certifier_proofs.o \
-$(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-print_signed_claim_obj= $(O)/print_signed_claim.o $(O)/support.o $(O)/certifier.o $(O)/certifier_proofs.o \
-$(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-package_claims_obj= $(O)/package_claims.o $(O)/support.o $(O)/certifier.o $(O)/certifier_proofs.o \
-$(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-print_packaged_claims_obj= $(O)/print_packaged_claims.o $(O)/support.o $(O)/certifier.o $(O)/certifier_proofs.o \
-$(O)/certifier.pb.o $(O)/simulated_enclave.o $(O)/application_enclave.o
+common_objs = $(O)/support.o $(O)/certifier.o $(O)/certifier_proofs.o \
```

Comment:
Refactored out common list of `.o`s into this symbol so the rest of the flow is easier to follow.